.github/workflows/lint.yml

```diff
@@ -556,6 +556,10 @@ jobs:
         uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
         with:
           submodules: false
+      - name: Checkout rest of history and master
+        env:
+          HASH: ${{ (github.event_name == 'pull_request' && github.event.pull_request.head.sha) || github.sha }}
+        run: git fetch --no-tags --prune --no-recurse-submodules --unshallow origin "${HASH}" master
```

Comment:
Would it make sense to make these repeated code blocks just composite actions?

Reply:
+1 could these be rolled into the checkout pytorch composite action?